apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/LoggingUtils.java

```diff
@@ -0,0 +1,38 @@
+package gov.cms.bfd.server.war.commons;
+
+import gov.cms.bfd.server.sharedutils.BfdMDC;
+import java.util.Arrays;
+import java.util.stream.Collectors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/** A set of methods for various logging purposes i.e. MDC */
+public class LoggingUtils {
+  private static final Logger LOGGER = LoggerFactory.getLogger(LoggingUtils.class);
+
+  /**
+   * Output list of benefificiary IDs to MDC logging
+   *
+   * @param beneIds the {@link Long} of beneficiary IDs top log
+   */
+  public static void logBeneIdToMdc(Long... beneIds) {
+    if (beneIds.length > 0) {
+      String beneIdEntry =
+          Arrays.stream(beneIds).map(String::valueOf).collect(Collectors.joining(", "));
+      BfdMDC.put("bene_id", beneIdEntry);
+    }
+  }
+
+  /**
+   * ensure a valid Long type benefificiary ID to log to MDC
```

Comment:
Suggestion for rewording:

Log a beneficiary ID to the BfdMDC under the 'bene_id' key if the ID supplied can be parsed as a long.

Reply:
Fixed for both